src/App.jsx

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} date={article.date} text={article.text} />
```

Comment:
можно было "разобрать" article.     {...article}

Reply:
Remember what we do when we have more than 3 attributes in a line, we put each of them on a new line
```suggestion
  <Article 
    title={article.title} 
    date={article.date} 
    text={article.text} 
  />
```